### ISSUE TITLE
need to add the curl option --spoof-edge-ip.

### ISSUE THREAD

**Issue by isss802** [akamai/cli-diagnostics#12]
In the API, spoofEdgeIp can be set.
https://techdocs.akamai.com/edge-diagnostics/reference/post-curl

However, I don't see that option in the CLI.
I think we need to add the feature.

```
akamai -s default diagnostics curl --help

Flags:
  -l, --client-location string       Optional. Unique identifier for an edge server location closest to your users. To get the value, run the 'edge-locations' command. Provide either this flag or '--edge-server-ip'.
  -e, --edge-server-ip string        Optional. IP of an edge server to use to run the command. If you don't know if an IP is the edge IP, run the 'verify-ip' command. Provide either this flag or '--client-location'.
  -i, --ip-version string            Optional. IP version to use to run the command, either 'IPV4' or 'IPV6'. By default set to 'IPV4'. (default "IPV4")
  -H, --request-header stringArray   Optional. Customized headers for the cURL request in the format 'header: value'. You can provide multiple values.
  -r, --run-from-site-shield-map     Optional. Runs the command from Site Shield.
  -h, --help                         help for curl
```

For example, this would allow curl to staging.

```
akamai -s default diagnostics curl https://test.fun --client-location tokyo-13-japan --spoof-edge-ip 23.50.62.33
```
